src/rt64lib/shaders/Tracer.hlsl

```diff
@@ -276,6 +276,7 @@ void FullShadeFromGBuffers(uint hitCount, float3 rayOrigin, float3 rayDirection,
 	float4 resColor = float4(0, 0, 0, 1);
 	float4 finalAlbedo = float4(0.0f, 0.0f, 0.0f, 0.0f);
 	float4 finalNormal = float4(0.0f, 0.0f, 0.0f, 0.0f);
+	half finalSpecular = half(0.0h);
```

Comment:
Final specular is not necessary since it doesn't get stored into one of the denoiser buffers.